### PR TITLE
Reformat codebase with Eslint

### PR DESCRIPTION
**Pull request by JakubFrejlach**
This PR:
* add the new linter rule from stylistic plugin
* reformats the whole codebase to comply with the Eslint rules
* adds lint check for `src` directory on build

Closes OSIDB-2204

### CODE REVIEW

**Comment by JakubFrejlach** [comment]
This one is a huge one but merging that ASAP will probably make our lives easier :smile:

File: src/components/ExampleForm.vue [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable -->
```
Comment (C-Valen) [inline src/components/ExampleForm.vue:1]: Eslint disabled

File: src/components/ExampleValidatedForm.vue [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable -->
```
Comment (C-Valen) [inline src/components/ExampleValidatedForm.vue:1]: Eslint disabled

**Review by C-Valen — APPROVED** [review]
LGTM, nice and clean linting config.

Just to mention there are some components with linting explicitly disabled:
src/components/BootstrapLabelInputExample.vue
src/components/ExampleValidatedForm.vue
src/components/ExampleForm.vue

I think if these will be maintained it might be good to lint them.

File: src/components/BootstrapLabelInputExample.vue [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable -->
```
[the remark this answers is not in the document]
Comment (JakubFrejlach) [reply src/components/BootstrapLabelInputExample.vue:1]: I am actually not sure about this one, @d10n @superbuggy any thoughts on this (and the two additional below)? These files feels to me like PoC files with a lot of changes and "furniture moving" still going on, so I was not sure whether strictly lint them.

File: src/components/ExampleForm.vue (hunk 1 above)
Comment (JakubFrejlach) [reply src/components/ExampleForm.vue:1]: Will be resolved the same way as we are going to agree on the case above.

File: src/components/ExampleValidatedForm.vue (hunk 2 above)
Comment (JakubFrejlach) [reply src/components/ExampleValidatedForm.vue:1]: Will be resolved the same way as we are going to agree on the case above.

**Review by superbuggy — APPROVED** [review]
Looks good. If resolving the merge conflicts is too annoying, you can rebase the commits on master

**Pull request merged**